openpype/hosts/nuke/api/lib.py

```diff
@@ -3423,3 +3423,55 @@ def create_viewer_profile_string(viewer, display=None, path_like=False):
     if path_like:
         return "{}/{}".format(display, viewer)
     return "{} ({})".format(viewer, display)
+
```

Comment:
This is not quite related to this PR but @jakubjezek001 probably can answer you this

Reply:
Not yet, but yes, this can be a global abstraction for future - If you want the Issue is yours @fabiaserra   ;)

Reply:
Admittedly the abstraction is more of an issue with the host DCC (especially with nuke apparently throwing this around) and Maya also doing it like that in the past. With OCIO v2 becoming more mainstream I feel like almost all APIs are updating with it and start supporting actual separate `display`  and `view` arguments/settings for things instead of requiring to combine them into a string. Hopefully this 'workaround' is just temporary and future versions of apps actually just supporting display separate from view.

Anyway, looking at this now. Are we sure this is correct?
```python
        return "{}/{}".format(display, viewer)
    return "{} ({})".format(viewer, display)
```
Did they reverse / swap around the arguments? 😢 That must make for quite some confusion.

Reply:
Oh sorry I might have confused you by the placing where I put the comment, I meant the functions actually added on this PR that simply remove frame hashes on filename `get_head_filename_without_hashes` and `get_filenames_without_hash`

Reply:
there's a lot of opportunities of creating better generic python util modules that we could reuse throughout hosts but the argument is generally "that needs a bigger refactor and it's outside of this PR"... so we just keep growing the codebase and duplicating the same utility functions with slight modifications over and over. I understand it's not always easy to do that but just wanted to raise it